index.js

```diff
@@ -1,9 +1,7 @@
 'use strict';
-var stringWidth = require('string-width');
+const stringWidth = require('string-width');
 
-module.exports = function (str) {
-	return Math.max.apply(null, str.split('\n').map(function (x) {
-		return stringWidth(x);
-	}));
+module.exports = str => {
+	return Math.max.apply(null, str.split('\n').map(x => stringWidth(x)));
```

Comment:
Should return this directly IMO without the curly brackets.

Reply:
Looking at this again makes me realize I can just do `.map(stringWidth)`.  Not sure if turbofan would optimize this anyways, but it seems like it will have map call stringWidth directly instead of calling a function that calls stringWidth.